routing/routing_integration_tests/get_altitude_test.cpp

```diff
@@ -24,14 +26,29 @@ namespace
 {
 using namespace feature;
 
+LocalCountryFile GetLocalCountryFileByCountryId(string const & countryId)
```

Comment:
Is it possible to use CountryFile instead of raw string here?

Reply:
Sure